src/pages/devtools/themes/natsuiro/js/Shipbox.js

```diff
@@ -260,8 +260,9 @@ KC3改 Ship Box for Natsuiro theme
 	---------------------------------------------------*/
 	KC3NatsuiroShipbox.prototype.showHP = function(){
 		// HP text
-		$(".ship_hp_cur", this.element).text( this.shipData.hp[0] );
+		$(".ship_hp_pred", this.element).text( this.shipData.hp[0] );
 		$(".ship_hp_max", this.element).text( "/"+this.shipData.hp[1] );
+		$(".ship_hp_cur", this.element).hide();

```

Comment:
i can't understand why `ship_hp_cur` element is removed in html but still have to hide it... and not match with your css and screenshot... seems some codes are missing?
suggest: don't use `ship_hp_pred`, still name it as `ship_hp_cur`, because when prediction option is off, the naming will make less confusing.